### ISSUE TITLE
Issue with FileConverter

### ISSUE THREAD

**Issue by Arthfael** [OpenMS/OpenMS#2980]
Now, I'm pretty new to OpenMS, so this may be a non-issue, please pardon me if this is silly.

I have some mzXML files I would like to convert to a MaxQuant friendly mzXML format. I tried to use FileConverter.exe and set option -force_MaxQuant_compatibility , however it turns out the version of the tool I have (2.1) does not recognize that option. If I search for the documentation online, the version that mentions that option is 2.2.
Can someone please tell me how I can update to that version? Do I need to install some beta or pre-release version?

**Comment by jpfeuffer** [comment]
Hi,
OpenMS 2.2 is not available as an installer.
However you can try to test our release candidate (=beta) for 2.3 here:
https://abibuilder.informatik.uni-tuebingen.de/archive/openms/OpenMSInstaller/RC/

If you just use the FileConverter the beta should be okay. However if you want to use the whole suite in production, you should reinstall as soon as we release it officially (hopefully next week).

**Comment by Arthfael** [comment]
Thank you very much for your help. I have just installed it, and tried
running it from command line and this is what I got:


*D:\test data\test>"C:\Program Files\OpenMS-2.3.0\bin\FileConverter.exe"
-in MyFile.mgf -out MyFile.mzXML -force_MaxQuant_compatibility
-write_scan_index trueAccess is denied.*

I then tried again running cmd as Administrator, and got the same error,
with an additional popup saying the application isn't a valid win32
application. The computer runs Windows 7 SP1 (64bit). Do you have an idea
what may be happening?

Also, note that I had to specify the exact path to the program because
there is also the Thermo FileConverter.exe installed on the system, which
has the exact same name and gets opened by default if I simply type
FileConverter in cmd. Since this may affect many other users, it may be a
good idea to create an alternative name through which people can also run
the application without having to deal with the ambiguity.

One more thing: The files I am dealing with are output from DIA Umpire
(pseudo-MSMS). These can be searched with diverse search engines, but since
I primarily use MaxQuant I was interested to see whether, with the correct
formatting, MQ can be tricked into performing a search on these too. This
would mean running MQ on originally DIA data, but processed to mimic DDA.
Is this something that in theory would be feasible, or is it just absurd? I
am afraid I do not know enough about the mzXML format to know whether this
makes sense.

Best regards,

Armel

**Comment by cbielow** [comment]
Hi Armel,

the error you are getting is probably due to the Thermo FileConverter.exe: you are executing the OpenMS2.3 fileconverter using a absolute filename but from a "random" path as working directory, but with other (probably conflicting) Thermo-packaged dll's on the PATH.
Try to 'cd' into the fileconverter directory and just use 'fileconverter.exe' to invoke it, using relative or absulte paths to you input and output data.
This should work.
If you want a more convenient method, use the TOPPAS Gui (shipped with OpenMS as TOPPAS.exe - available as shortcut from start program menu) and build a 3 node pipeline (input files -> fileconverter -> output files). TOPPAS will take care to call the correct executables.

cheers
Chris

**Comment by cbielow** [comment]
PS: the DIA UMPIRE -> Fileconverter -> maxQuant approach could work. Let us know what happened :)

**Comment by Arthfael** [comment]
Dear Chris,

It is working now. I first tried your cmd suggestion, and it failed... then
tried TOPPAS, which crashed when I loaded the node. This made me think that
probably the installation was broken, and it turned out to be true! Having
reinstalled, everything is now working, I did manage to get mzXML files
with the force MaxQuant compatibility option set to true. So that is one
hurdle cleared, thank you!

The output files, however, do still cause MaxQuant (version 1.6.0.13) to
crash after a few seconds. Oh well. I really liked the idea of using DIA
Umpire to make MaxQuant run on DIA data, so will be happy to test any other
suggestion you may have. Regardless, I will also test some of the other
search engines available in OpenMS.

Thank you very much for your help and have a nice week end!

Kind regards,

Armel

PS: I also have a question about the CompNovo tool: the help says it was
not tested properly and may not work properly. Will this be fixed at some
point? How cautious should I be when using it?

**Comment by Arthfael** [comment]
Aaaahhh, of course, the files I have only contain the pseudo-MS2 spectra,
no MS1 at all. I may have to find a way to also append the MS1 spectra for
MQ to be able to (maybe) deal with them.

**Comment by timosachsenberg** [comment]
Currently, we had only a small number of requests regarding the deNovo sequencing tool so it is not really well maintained. My advice would be to use a different tool at this point.

**Comment by Arthfael** [comment]
This is strange, i would think there would be more demand for de novo
sequencing analysis. Oh well, I will see what else I can find.

I have tried again using MaxQuant. This time I used msconvert to create a
file that was a merge of MS1 extracted from the original DIA mzXML file, as
well as the pseudo-MS2 generated by DIA Umpire (Q1, 2 and 3). This file was
then converted to a MaxQuant-compatible mzXML using FileConverter as above,
but this time too something must be missing as MaxQuant still crashes
quickly during file testing. I think I will have to give it up, I do not
have the knowledge to figure out what else I need to do.

Best,

Armel



<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#m_3661306466465098258_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Mon, Oct 30, 2017 at 11:50 AM, Timo Sachsenberg <notifications@github.com
